templates/init-d.ejs

```diff
@@ -14,7 +14,11 @@
 
 dir="<%- workingDirectory %>"
 user="<%- uid %>"
-cmd="<% Object.keys(env).forEach(function(key) {%><%- key %>=<%- env[key] %> <% }); %><%- nodeBin %> <%- startScript %><% flatArgs.forEach(function(arg) { %> <%- arg %><% }); %>"
+<% Object.keys(env).forEach(function (key) { %>
+<%- key %>=<%- env[key] %>
```

Comment:
That's a `sh` vs `bash` thing. `bash` allows it on one line; other bourne shells require the two-line form.